webassembly/main.js

```diff
@@ -73,19 +73,40 @@ var manifest_link = url.searchParams.get("manifest");
 var ram_val = url.searchParams.get("ram");
 var cpu_val = url.searchParams.get("cpu");
 var mhz_val = url.searchParams.get("mhz");
+var lang_val = url.searchParams.get("keymap");
+var diag = url.searchParams.get("longpwron");
+var wide = url.searchParams.get("widescreen");
+var capture = url.searchParams.get("capture");
+var midline = url.searchParams.get("midlineeffects");
 
 var emuArguments = ['-keymap', lang, '-rtc'];
 
 if (ram_val) {
     emuArguments.push('-ram', ram_val);
 }
 if (cpu_val) {
-    if (cpu_val == 'c816')
+    if (cpu_val == 'c816') {
         emuArguments.push('-c816');
+    }
```

Comment:
Just one point here, it's possible that 816 will become the default later, so perhaps this should check for c02 in cpu_val and if it is set, add `-c02` to the emu args.

This will have to be done in a new PR.